app/assets/javascripts/application.js

```diff
@@ -7,9 +7,22 @@
 //= require lookup_keys
 
 $(function() {
-  $(document).trigger('ContentLoad');
+  if(window.allJsLoaded){
+    $(document).trigger('ContentLoad');
+  }
+  else {
+  $(document).on('loadJS', function() {
```

Comment:
not sure if it's related to this change, but when moving from legacy UI page to a React page, on every API `get` request I see the following error:

![image](https://github.com/theforeman/foreman/assets/26363699/d3eec3d4-306a-4fb4-b06d-646fe2028fc1)


I believe that `Axios` is using the DOM to store some of the request properties, and when trying to access an element it fails to find it, could be some race condition.

overall it doesn't seem to affect the pages functionality.

Reply:
I cant reproduce this, which pages are you moving from/to?

Reply:
got fixed for me in the latest commit and @theforeman/vendor v12.2.3